docs/source/transaction_family_specifications/supplychain_transaction_family.rst

```diff
@@ -0,0 +1,255 @@
+***************************************
+Supply Chain Transaction Family
+***************************************
+
+Overview
+=========
+
+The Supplychain Transaction Processor(SCTP) provides a mechanism for
+storing records of asset location and ownership.
+
+State
+=====
+
+This section describes in detail how SCTP objects are stored and addressed in the global state.
+
+SCTP stores the following data objects:
+
+Agent
+-----
+
+Agents represent owners or holders of goods in the system. All agents must be registered in the system before they can be used.
+
+.. code-block:: javascript
+
+    {
+        Name: "name", // The name of the agent
+        Type: "type", // Agent type, domain specific value
+        Url: // The web address of the agent as a fully qualified URL
+        OwnRecords: [], // list of all records(by address) this agent owns
+        HoldRecords: [], // list of all records(by address) this agent holds
+        OpenApplications: [], // list of all applications(by records address) this agent owns
+        AcceptedApplications: [] // list of all records this agent owns
+    }
+
+For "HoldRecords", a value 1 indicates there is an open application against the record, 0 means no open application.
```

Comment:
I'm not clear on what this means. It looks like HoldRecords is a list in the above json.